### DISCUSSION TITLE
Publish ke shiny tidak bisa dengan reason token expired

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#257] [Q&A] [answered]
Halo,

Mohon bantuannya, mau publish dashboard ke shiny terbentur dengan token expired :
![image](https://user-images.githubusercontent.com/107389614/183070086-ea78bb96-039e-4d4d-9754-9c46661c6d76.png)

terima kasih

**Comment by RanggaGemilang** [comment]
Haloo saya jg mengalami kendala yang sama, apakah harus menggunakan token dan slot baru untuk re-deploy shinyapps yang sudah ada?

**Comment by RanggaGemilang** [comment, marked as answer]
halo ini sudah solved ya ternyata memang ada error dari pihak RStudio nya https://community.rstudio.com/t/how-can-i-do-with-expiredtokenexception-error/144032/12
